common/src/main/java/net/dodogang/plume/mixin/client/CreativeInventoryScreenMixin.java

```diff
@@ -62,10 +67,18 @@ public void render(MatrixStack matrixStack, int mouseX, int mouseY, float delta,
         });
     }
 
-    @Inject(method = "renderTabIcon", at = @At(value = "INVOKE", target = "Lnet/minecraft/client/gui/screen/ingame/CreativeInventoryScreen;drawTexture(Lnet/minecraft/client/util/math/MatrixStack;IIIIII)V", shift = At.Shift.BEFORE))
-    protected void renderTabIcon(MatrixStack matrixStack, ItemGroup itemGroup, CallbackInfo ci) {
+    @Inject(
+        method = "renderTabIcon",
+        at     = @At(
+            value = "INVOKE",
+            target =
+                "Lnet/minecraft/client/gui/screen/ingame/CreativeInventoryScreen;drawTexture(Lnet/minecraft/client/util/math/MatrixStack;IIIIII)V",
+            shift = At.Shift.BEFORE))
+    protected void
+    renderTabIcon(MatrixStack matrixStack, ItemGroup itemGroup, CallbackInfo ci) {
```

Comment:
I don't know why it put the `protected void` on a different line here.

Reply:
I also find cringe